examples/docker/video-rag/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
```

Comment:
ploomber cloud does not support docker-compose, you'll need to modify your dockerfile so it runs elasticsearch instead of using this